service/orchestrator/certificates.go

```diff
@@ -0,0 +1,172 @@
+package orchestrator
+
+import (
+	"context"
+	"errors"
+
+	"clouditor.io/clouditor/api/orchestrator"
+	"clouditor.io/clouditor/internal/logging"
+	"clouditor.io/clouditor/persistence"
+	"clouditor.io/clouditor/service"
+
+	"github.com/sirupsen/logrus"
+	"golang.org/x/exp/slices"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+const certificationNotFoundErrorMessage = "certificate not found"
```

Comment:
Shouldn't this be `ErrCertificateNotFound` or something?

Reply:
It was a string which i wrapped into gRPC errors and it is not an error itself. But after thinking about it, I now use a gRPC error directly.
I'm just not sure if such api error messages (=grpc errors) are better kept in the api package in general. Since it only used here, I leave it here.